pages/react-canvas-components.mdx

```diff
@@ -0,0 +1,256 @@
+import { RandomHexagons } from '../components/hexagons/RandomHexagons';
+import { EditOnCodeSandbox } from '../components/EditOnCodeSandbox';
+import { css } from '@emotion/core';
+
+export const meta = {
+  title: 'React + Canvas = 💜',
+  description: 'an untold love story',
+  date: 'January 12, 2019',
+};
+
+The `<canvas/>` HTML element can be used to draw graphics with a finer control than the usual DOM or SVG. But with React, trying to draw on a canvas is not intuitive as their interfaces are quite different. With React, each component own their node contrary to canvas where there is only one shared node, and we draw in it. Let's see how we can make a canvas visualization with React components !
```

Comment:
- "each component *owns* their node, as opposed to canvas ..."
- ".. one shared node that we can use for drawing"